### PR TITLE
[TS] LPS-155431

### PR DESCRIPTION
**Pull request by yuhai**
@dantewang, I didn't change the fix logic, only added SF commits.

cc [jorgediaz-lr](https://github.com/jorgediaz-lr)

### CODE REVIEW

File: modules/apps/static/portal-osgi-web/portal-osgi-web-portlet-container-test/src/testIntegration/java/com/liferay/portal/osgi/web/portlet/container/embedded/test/EmbeddedPortletWhenEmbeddingPortletInLayoutTemplateTest.java [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+
+		Assert.assertEquals(
+			Arrays.asList(
+				"TEST_COLUMN-1_PORTLET_2", "TEST_COLUMN-1_PORTLET_1",
+				"TEST_EMBEDDED_PORTLET_3", "TEST_EMBEDDED_PORTLET_2",
+				"TEST_EMBEDDED_PORTLET_1"),
+			renderedPortlets);
+	}
+
+	private HttpServletRequest _getHttpServletRequest() throws Exception {
```
Comment (dantewang) [inline modules/apps/static/portal-osgi-web/portal-osgi-web-portlet-container-test/src/testIntegration/java/com/liferay/portal/osgi/web/portlet/container/embedded/test/EmbeddedPortletWhenEmbeddingPortletInLayoutTemplateTest.java:118]: @yuhai This method is only used once, please inline. cc @jorgediaz-lr

**Review by dantewang — CHANGES_REQUESTED** [review]
@yuhai @jorgediaz-lr The PR looks good to me, just need an additional SF as I commented. Thanks!

**Comment by yuhai** [comment]
send it to https://github.com/liferay-core-infra/liferay-portal/pull/980

**Comment by yuhai** [comment]
ci:stop

**Pull request closed without merging**